src/Crypt/OpenSSL.php

```diff
@@ -21,8 +21,7 @@ class OpenSSL implements Crypt
 		$secureKey = substr($secureKey, 0, 24);
 
 		// Fill bytes
-		$fullFill = 8 - strlen($data) % 8;
```

Comment:
@f3l1x A problem was here. It can be padded by +1 empty byte although it wasn't necessary.